CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Node.js Agent for Google Cloud Debug ChangeLog
 
+## 2017-10-23, Version 2.2.0 (Beta), @dominickramer
+
+This release includes experimental support for the new [V8 Inspector Protocol](https://chromedevtools.github.io/debugger-protocol-viewer/v8/) ([#329](https://github.com/GoogleCloudPlatform/cloud-debug-nodejs/pull/329)) and fixes an issue with setting snapshots on lines that cannot directly have snapshots set, such as comments ([#330](https://github.com/GoogleCloudPlatform/cloud-debug-nodejs/pull/330)).
+
+The Stackdriver Debugger will use the V8 Inspector Protocol if and only if the `GCLOUD_USE_INSPECTOR` environment variable is set and the running version of Node supports the inspector protocol (Node 8+).
+
+If the `GCLOUD_USE_INSPECTOR` environment variable is set, but the running version of Node does not support the inspector protocol, the agent will fall back to the legacy debugger protocol and a warning message will be logged.
+
+### Commits
+* [[`c15872df2c`](https://github.com/GoogleCloudPlatform/cloud-debug-nodejs/commit/c15872df2c)] - Inspector only enabled when specified (#343) (Michael Gao)
```

Comment:
Not actionable here, but going forward it would be good to make sure we are using conventional commit style at landing time for all commits.